streaming.go

```diff
@@ -65,8 +65,8 @@ func newStreamingState(
 }
 
 type sendReceiver interface {
-	Receive(any) error
-	Send(any) error
+	Receive(msg any) error
+	Send(msg any) error
```

Comment:
I would probably just exclude the `inamedparam` checker in this repo, too.

Reply:
Disabled the lint.